.github/workflows/deploy.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - uses: svenstaro/upload-release-action@v2
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: ./neuralegion-bright-cli-*
+          file: ./brightsec-cli-*
```

Comment:
You have to change the npm scope in the `publish` job as well. Also,  should we consider changing the Docker Hub repository (e.g. `neuralegion/repeater` vs `brightsec/repeater` or `brightsec/cli`)?